client/gefyra/cluster/utils.py

```diff
@@ -55,3 +55,50 @@ def get_env_from_pod_container(
     raise RuntimeError(
         f"Failed to get env from pod {pod_name} in namespace {namespace} after {retries} tries."
     )
+
+
+def get_container(pod, container_name: str):
+    for container in pod.spec.containers:
+        if container.name == container_name:
+            return container
+    raise RuntimeError(f"Container {container_name} not found.")
+
+
+def get_container_image(pod, container_name: str):
+    container = get_container(pod, container_name=container_name)
+    if container.image:
+        return container.image
+    raise RuntimeError(f"Container {container_name} image could not be determined.")
+
+
+def get_container_command(pod, container_name: str):
```

Comment:
the `args` attribute is missing here and must be considered, e.g.
```yaml
[...]
command: ["/bin/bash", "-c"]
args: ["python manage.py serve"]
```

Reply:
Good point, thank you!

Reply:
Changed the implementation.